charts/presidio/templates/tester-job-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+{{ $fullname := include "presidio.tester.fullname" . }}
```

Comment:
add if statement, to make sure this deployment is only happening on explicit scenario